CHANGELOG.md

```diff
@@ -3,9 +3,9 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## 1.0.0dev
+## 2.0.0 - 2024/02/14
 
-Development release of phac-nml/snvphylnfc.
+The initial release of phac-nml/snvphylnfc.
```

Comment:
Could you make this a bullet point?

Also, could you include more information here. In particular, that this is a Nextflow version of SNVPhyl following the nf-core pipeline standards. And that this is referred to as version 2 since it changes how SNVPhyl is run (Nextflow as opposed to Galaxy).

Also, in order to follow the [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) structure, this should all be put under a subheading `### Changed`.

Reply:
Updated in cb4a47f1b8e15d2847d76eba810883e1b286fb06